src/django/iow/settings.py

```diff
@@ -0,0 +1,237 @@
+"""
+
+import os
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+#TODO: move this to .env
+
+SECRET_KEY = os.getenv("DJANGO_SECRET_KEY", "secret")
+
+ENVIRONMENT = os.getenv("DJANGO_ENV", "Development")
+VALID_ENVIRONMENTS = ("Production", "Staging", "Development")
+if ENVIRONMENT not in VALID_ENVIRONMENTS:
+    raise ImproperlyConfigured(
+        "Invalid ENVIRONMENT provided, must be one of {}".format(VALID_ENVIRONMENTS)
+    )
+
+LOGLEVEL = os.getenv("DJANGO_LOG_LEVEL", "INFO")
+
+GIT_COMMIT = os.getenv("GIT_COMMIT", "UNKNOWN")
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = ENVIRONMENT == "Development"
+
+ALLOWED_HOSTS = [
+    "localhost",
+]
+
+if ENVIRONMENT == "Development":
+    # Services within the bridge network access Django by resolving its
+    # container name. Also, some folks develop on remote workstations.
+    ALLOWED_HOSTS.extend(os.getenv("DJANGO_ALLOWED_HOSTS", "localhost").split(","))
+
+if ENVIRONMENT in ["Production", "Staging"]:
+    # The Elastic Load Balancer HTTP health check will use the target
+    # instance's private IP address for the Host header.
+    #
+    # The following steps look up the current instance's private IP address
+    # (via the ECS container metadata URI) and add it to the Django
+    # ALLOWED_HOSTS configuration so that health checks pass.
+    response = requests.get(os.getenv("ECS_CONTAINER_METADATA_URI"))
+    if response.ok:
+        container = response.json()
+        for network in container["Networks"]:
+            for addr in network["IPv4Addresses"]:
+                ALLOWED_HOSTS.append(addr)
+    else:
+        raise ImproperlyConfigured("Unable to fetch instance metadata")
+
+    # Ensure that Django can service requests originating from our public hosted
+    # zone.
+    R53_PUBLIC_HOSTED_ZONE = os.getenv("R53_PUBLIC_HOSTED_ZONE")
+    if R53_PUBLIC_HOSTED_ZONE == None:
+        raise ImproperlyConfigured("Empty R53_PUBLIC_HOSTED_ZONE provided")
+    ALLOWED_HOSTS.append(".{}".format(os.getenv("R53_PUBLIC_HOSTED_ZONE")))
+
+    # Ensure Django knows to determine whether an inbound request was made over
+    # HTTPS by the ALB's HTTP_X_FORWARDED_PROTO header.
+    SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
+
+    # Session Security
+    SESSION_COOKIE_SECURE = True
+    CSRF_COOKIE_SECURE = True
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    "whitenoise.runserver_nostatic",
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    "django.contrib.gis",
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    # 'django.contrib.staticfiles',
```

Comment:
Is this commented out because of the use of Django-SPA?

Reply:
I tried setting up the static files stuff, but was getting errors, so I figured I'd leave it commented out for now, and circle back to it later since we're not trying to serve any static content right now. I'll open an issue to fix this

Reply:
Sounds good. Thanks!